src/Command/DataFlow/DataFlowTimeoutJobsCommand.php

```diff
@@ -42,8 +42,7 @@ public function configure()
     public function execute(InputInterface $input, OutputInterface $output)
     {
         $timeoutThreshold = $input->getOption('timeout-threshold');
-        $timeoutThresholdIsInteger = (int) $timeoutThreshold === $timeoutThreshold && $timeoutThreshold > 0;
-        if (!$timeoutThresholdIsInteger) {
+        if (false === filter_var($timeoutThreshold, FILTER_VALIDATE_INT)) {
```

Comment:
This will allow `0` and negative values. We should probably add additional checks to make sure that `timeoutThreshold` actually makes sense.